apps/block_scout_web/lib/block_scout_web/views/layout_view.ex

```diff
@@ -114,8 +114,11 @@ defmodule BlockScoutWeb.LayoutView do
           nil
 
         release_link_env_var == "" || release_link_env_var == nil ->
-          "https://github.com/celo-org/blockscout/releases/tag/" <> version
-
+          if String.length(version) == 40 do 
```

Comment:
I'm not so sure that this is the best way of detecting a git hash, but I can't think of anything else right now...